docs/postgres.md

```diff
@@ -63,6 +63,20 @@ You may need to enable password authentication so `synapse_user` can
 connect to the database. See
 <https://www.postgresql.org/docs/11/auth-pg-hba-conf.html>.
 
+### Fixing incorrect `COLLATE` or `CTYPE`
+
+Synapse will refuse to set up a new database if it has the wrong values of
+`COLLATE` and `CTYPE` set, and will log warnings on existing databases. Using
+different locales can cause issues if the locale library is updated from
+underneath the database, or if a different version of the locale is used on any
+replicas.
+
+The safest way to fix the issue is to take a dump and recreate the database with
+the correct `COLLATE` and `CTYPE` parameters (as per
+[docs/postgres.md](docs/postgres.md)). It is also possible to change the
+parameters on a live database and run a `REINDEX` on the entire database,
+however extreme care must be taken to avoid database corruption.
```

Comment:
won't either of these methods fail until you take steps to remove duplicate rows?

Reply:
Oh, true. TBH I'm not really sure what to suggest they do other than a bland "delete them maybe?"

Reply:
+1 to "delete them maybe"

btw the secret `ctid` pseudo-column can be useful for distinguishing similar rows